### DISCUSSION TITLE
Will Have DevillutionX On Xbox 360 In The Future?

### DISCUSSION THREAD

**Discussion by ZarkTye** [diasurgical/DevilutionX#3385] [Q&A] [answered]

**Comment by StephenCWills** [comment, marked as answer]
See: https://github.com/diasurgical/devilutionX/discussions/2450

FYI, to my knowledge, we don't currently have a maintainer for an Xbox 360 port so I don't think there are any concrete plans to add it.

**Reply by AJenbo** [reply]
I have an Xbox 360 and some vague plans of getting it modded and try my hand at porting to it

**Reply by AJenbo** [reply]
We will need this if if we ever start looking into making a port for Xbox 360: https://github.com/LibXenonProject/libSDLXenon

**Reply by ZarkTye** [reply]
I Have Rgh Xbox 360

**Reply by AJenbo** [reply]
You are more then welcome to start working on a port :)

**Reply by ZarkTye** [reply]
I Not Will Make A Port,And If You Not Want It's ok

**Reply by qndel** [reply]
Why Capitalize Every Word Like A PoKeMoN? 😕

**Reply by ZarkTye** [reply]
I Use Android

**Reply by Kristopian** [reply]
@ZarkTye It is possible to play DevilutionX on Xbox 360 hardware (I don't think you can run more than one OS on one hard drive), using ArchLinux, it is risky to actually run a different OS on Xbox 360 Slightly easier if the Xbox is Jtagged or a RGH-d (I think your's is, but correct me if i'm wrong). While you can run DevilutionX on Xbox, it will run as well as a actual Homebrew app, but running a different OS on a Xbox 360 is a bit less reliable and you really need to know what you are doing or you might brick or permanently damage the Xbox 360. I may have missed a more obvious solution so feel free to yell at me in a civilized, mature way, I don't think I have to tell you all this, but i have seen people lose their @!?# over me using a Iphone 5s instead of the more modern phones(and other stupid stuff).
